app/assets/javascripts/discourse/app/widgets/topic-post-visited-line.js

```diff
@@ -3,16 +3,12 @@ import { createWidget } from "discourse/widgets/widget";
 import { h } from "virtual-dom";
 
 export default createWidget("topic-post-visited-line", {
-  tagName: "div.topic-post-visited-line",
+  tagName: "div.small-action.topic-post-visited",
```

Comment:
The only thing that stands out to me here is if there is any JS theme components/plugin code that iterate through `small-action` posts and modify them. If not that's fine then!

Reply:
Honestly, I'm not sure but I guess only one way to find out.